CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v8.0.0]
+
+### Changed 
+- Double encoding values provided to the rql builder is enabled by default
+- Example query: `rqlBuilder().eq(name, '< value >').build()`
+  - Current behaviour: `?eq(name,%253C%2520value%2520%253E)`
+  - Old behaviour: `?eq(name,< value >)`
```

Comment:
I belief it may be stressed more that this is a breaking change. I think we could urge developers to check their project for the use of encoding mechanisms like `encodeURIComponent()` before upgrading and/or setting the `rqlBuilder.doubleEncodeValues = false`